src/blocks/gist/components/gist.js

```diff
@@ -97,11 +106,13 @@ export default class Gist extends Component {
 				gistContent: gist.div
 			});
 		}
-
 		let gistScript = document.createElement('script');
 		gistScript.type = 'text/javascript';
 		gistScript.src = this._tranformedURL(gistCallback);
-		document.head.appendChild(gistScript);
+		gistScript.onerror = function (err) {
+		  _handleScriptError();
```

Comment:
I think this needs to be `this._handleScriptError()` otherwise the method is not being called.

Reply:
Disregard. Because we're inside a callback function `this` will reference the script element rather than the Gist class. I see that `const { _handleScriptError } = this;` is the workaround.